content/My_content/bulbasaur.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+  <title>Bulbasaur</title>
+  <meta name="author" content="Diego Wagner"/>
+  <meta name="description" content="The Bulbasaur Family"/>
+  <meta name="keywords" content="Pokemon,Bulbasaur,The Bulbasaur Family,Ivysaur, Venusaur"/>
+</head>
+<body>
+<div>
+  <header>
+    <ul>
+      <li><a href="bulbasaur.html">The Bulbasaur Family</a></li>
+      <li><a href="charmander.html">The Charmander Family</a></li>
+      <li><a href="squirtle.html">The Squirtle Family</a></li>
+    </ul>
+</header>
+</div>
+<div>
+<h1>The Bulbasaur Family</h1>
+<h2>Bulbasaur</h2>
+</div>
+<div>
+<table>
+  <tbody>
```

Comment:
Missing `thead`. If you are going to use a `table`, best to always include table headers. This is probably better suited as a `ul`.